net-analyzer/greenbone-security-assistant/greenbone-security-assistant-21.4.4.ebuild

```
@@ -0,0 +1,107 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+inherit cmake systemd toolchain-funcs
+
+MY_PN="gsad"
+
+DESCRIPTION="Greenbone Security Assistant"
+HOMEPAGE="https://www.greenbone.net/en/ https://github.com/greenbone/gsa"
+SRC_URI="https://github.com/greenbone/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2+"
+KEYWORDS="~amd64 ~x86"
+IUSE="extras"
+
+DEPEND="
+	acct-group/gvm
+	acct-user/gvm
+	dev-libs/libgcrypt:0=
+	dev-libs/libxml2
+	dev-libs/libxslt
+	>=net-analyzer/gvm-libs-21.4.1
+	net-libs/gnutls:=
+	net-libs/libmicrohttpd"
+
+RDEPEND="
+	${DEPEND}
+	~net-analyzer/greenbone-security-assistant-${PV}
+	>=net-analyzer/gvmd-21.4.1
+	net-analyzer/ospd-openvas"
+
+BDEPEND="
+	dev-python/polib
+	virtual/pkgconfig
+	extras? (
+		app-doc/doxygen[dot]
+		app-doc/xmltoman
+		app-text/htmldoc
+		sys-devel/gettext
+	)"
+
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+src_prepare() {
+	cmake_src_prepare
+	# QA-Fix | Remove !CLANG doxygen warnings for 9.0.0
+	if use extras; then
+		if ! tc-is-clang; then
+		   local f
+		   for f in doc/*.in
+		   do
+			sed -i \
+				-e "s*CLANG_ASSISTED_PARSING = NO*#CLANG_ASSISTED_PARSING = NO*g" \
+				-e "s*CLANG_OPTIONS*#CLANG_OPTIONS*g" \
+				"${f}" || die "couldn't disable CLANG parsing"
+		   done
+		fi
+	fi
+}
+
+src_configure() {
+	local mycmakeargs=(
+		"-DLOCALSTATEDIR=${EPREFIX}/var"
+		"-DSYSCONFDIR=${EPREFIX}/etc"
+		"-DSBINDIR=${EPREFIX}/usr/bin"
+		"-DDEFAULT_CONFIG_DIR=${EPREFIX}/etc/default"
+		"-DLOGROTATE_DIR=${EPREFIX}/etc/logrotate.d"
+		"-DGSAD_RUN_DIR=${EPREFIX}/run/gvmd"
```

I think this would create files under `/run` in the installation image and hence issue a QA warning. Instead we could probably do the same as in the `net-analyzer/gvm-libs` ebuild:

```suggestion
		"-DGSAD_RUN_DIR=${EPREFIX}/var/lib/gvmd"
```

and

```
keepdir /var/lib/gvmd
fowners -R gvm:gvm /var/lib/gvm
```

in `src_install`.